CMakeLists.txt

```diff
@@ -24,7 +24,7 @@ if (MSVC)
 
     add_compile_options(
         /bigobj       # large object file format
-        /permissive-  # strict C++
+        #/permissive-  # strict C++
```

Comment:
Is this change intentional?

Reply:
Yes because another Boost library shits the bed otherwise

Reply:
https://github.com/boostorg/range/issues/68